extensions/calendly/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "calendly",
+  "title": "Calendly",
+  "description": "Raycast commands to access Calendly instance",
+  "icon": "logo.png",
+  "author": "eluce2",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "calendly",
+      "title": "Calendly",
+      "subtitle": "Copy Meeting URL",
```

We do the command titles the other way around ;-) Usually we have some `<Verb> <Noun>` as command title and use the service as subtitle. How about something like this?

```suggestion
      "title": "Share Meeting Link",
      "subtitle": "Calendly",
```